### PR TITLE
Update redirects

### PR DESCRIPTION
**Pull request by filiph**

### CODE REVIEW

**Comment by filiph** [comment]
Actually, hold it. I want to sort the redirects and remove duplicates.

**Comment by filiph** [comment]
Ok now it's ready for review.

File: firebase.json [hunk 1]
```diff
@@ -16,43 +16,38 @@
...
-        "source": "/articles/numeric-computation/",
-        "destination": "/articles/dart-vm/numeric-computation",
+        "source": "/+lexicalscope",
+        "destination": "/guides/language/language-tour#lexical-scope",
         "type": 301
       },
       {
-        "source": "/articles/simd/",
-        "destination": "/articles/dart-vm/simd",
+        "source": "/+pub",
```
Comment (kevmoo) [inline firebase.json]: What does the `+` do here? Just curious.

Comment (kwalrath) [inline firebase.json]: IIRC, @sethladd created a bunch of /+ shortcuts. For example, near me I see an old poster for lexical scope. At the bottom of it is the URL dartlang.org/+lexicalscope.

File: firebase.json [hunk 2]
```diff
@@ -721,7 +726,7 @@
         "type": 301
       },
       {
-        "source": "/pub/tools/editor",
```
Comment (kwalrath) [inline firebase.json]: How do you know whether the original URL was editor/ or editor.html?

Comment (filiph) [inline firebase.json]: We don't :(. Were they different pages at some point?

Comment (kwalrath) [inline firebase.json]: I don't recall. I was just wondering if you had a trick.

**Pull request merged**